PlainSql.Migrations.Tool/PlainSql.Migrations.Tool.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <PackAsTool>true</PackAsTool>
+    <PackageOutputPath>./nupkg</PackageOutputPath>
+    <ToolCommandName>migrate</ToolCommandName>
```

Comment:
❓ is the tool name to generic?

Reply:
possibly but figured we'd want a one word name, open to alternatives :)